### PR TITLE
Card component added

### PR DESCRIPTION
**Pull request by ishapatell1**

### CODE REVIEW

File: Components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  }
+  .card-badge {
+    position: absolute;
+    background-color: var(--primary-color);
+    color: #eee;
+    padding: 1px;
+    top: 5px;
+    left: 0px;
+    border-top-right-radius: 4px;
+    border-bottom-right-radius: 4px;
```
Comment (aqsaaqeel) [inline Components/Card/card.css:25]: You can use rem here instead of px

**Review by aqsaaqeel — APPROVED** [review]

**Review by shivangisharma01 — APPROVED** [review]
The looks clean, good t merge

File: Components/Badge/badge.css [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
-@import url(../../css/color.css);
+@import url(/css/Allcss.css);
+
+ 
+ 
 
 .badge-container {
   position: relative;
+  margin-left: 1rem;
+  color: aliceblue;
```
Comment (muskan-gupta-0803) [inline Components/Badge/badge.css:10]: You can use color variable here. Or other way can be hex code.

**Review by muskan-gupta-0803 — APPROVED** [review]
Just that minor change.

Rest all looks good to merge.

File: Components/Card/card.css (hunk 1 above)
Comment (SukanyaSen530) [reply Components/Card/card.css:25]: Better not to use rem for less than 10px

**Pull request merged**